src/db/sysdb_ops.c

```diff
@@ -3952,6 +3991,8 @@ int sysdb_delete_user(struct sss_domain_info *domain,
         ret = sysdb_search_user_by_uid(tmp_ctx, domain, uid, NULL, &msg);
     }
     if (ret == EOK) {
+        struct ldb_dn *override_dn = NULL;
```

Comment:
I prefer to have variable declared at the beginning of the function. But there are examples for not doing this even in this function, so feel free to ignore this comment.